frontend/src/components/nodes/CumuativeStakeChart.tsx

```diff
@@ -0,0 +1,50 @@
+interface Props {
+  value: CummulativeStake;
+}
+
+interface CummulativeStake {
```

Comment:
```suggestion
interface CumulativeStake {
```

Also, rename the file to have a proper name "cumulative"